server/lib/pbench/indexer.py

```diff
@@ -1018,20 +1017,22 @@ def __init__(self, ptb):
         )
         self.json_dirs = None
         try:
-            self.json_dirs = ResultData.get_result_json_dirs(ptb)
+            self.json_dirs = ResultData._get_result_json_dirs(ptb)
         except KeyError:
-            self.json_dirs = None
+            pass
+        # Check to see if this is a "known" user benchmark.
+        self.user_benchmark = ResultData._known_user_benchmark(ptb)
 
     @staticmethod
-    def get_result_json_dirs(ptb):
+    def _get_result_json_dirs(ptb):
         """
         Fetch the list of directories containing result.json files for this
         experiment; return a list directory path names.
         """
         paths = [
-            x
-            for x in ptb.tb.getnames()
-            if (os.path.basename(x) == "result.json") and ptb.tb.getmember(x).isfile()
+            x.name
+            for x in ptb.members
+            if x.isfile() and (os.path.basename(x.name) == "result.json")
```

Comment:
missing colon?

Reply:
This is a list comprehension, so no colon needed.